packages/components/input/src/index.html

```diff
@@ -20,7 +20,24 @@
   </style>
 </head>
 <body style="margin: 0;">
-  <osds-input inline type="text" value="On Vous Héberge ?" disabled clearable></osds-input>
```

Comment:
Can you keep some test of input outside of a <form> element to ensure they can still be used?